{{cookiecutter.project_slug}}/Dockerfile

```diff
@@ -1,17 +1,19 @@
 FROM python:3.6-alpine
 
-ENV PYTHONUNBUFFERED 1
+ENV PYTHONUNBUFFERED=1
 
 ARG INSTALL_PATH=/app
 
+ENV PYTHONPATH=${INSTALL_PATH}
```

Comment:
It seems incorrect to add the install path to `PYTHONPATH`, but I realize this is necessary since we're not installing the python package locally. A couple of alternatives:

* Make a module out of tests, i.e. creating `tests/__init__.py`
* Move python source to `src/{{ project_slug }}` and `src/tests`, and set `ENV PYTHONPATH=${INSTALL_PATH}/src`

Thinking about it, I'm not a big fan of those solutions either so I'm fine with leaving it as it is too.

Reply:
I would perhaps lean towards adding `tests/__init__.py` being the simplest one, but I'll leave the choice up to you.

Reply:
How would that help? The problem is that the tests need to import the code modules. That means the `src` modules need to be on the `PYTHONPATH`.